packages/celotool/src/lib/eksportisto.ts

```diff
@@ -47,7 +46,7 @@ function buildContext(celoEnv: string): Context {
   const suffix = fetchEnvOrFallback(envVar.EKSPORTISTO_SUFFIX, '')
   const chartDir = chartDirForVersion[chartVersion]
   if (chartDir === undefined) {
-    throw new Error('version has to be 1, 2 or 3')
+    throw new Error('version has to be 2 or 3')
```

Comment:
would it be better to change the set of versions from (1, 2, 3) to (1, 2) instead of (2, 3)?

Reply:
It'd surely look nicer, but it would also mean changing celotool code that checks what chart version is being used, and that's trickier to test without re-deploying eksportisto-14/no_version. As soon as 14 is decommissioned I'd remove this chart version logic, as we'll then have a single chart version.